src/rpc/net.cpp

```diff
@@ -299,6 +300,25 @@ static RPCHelpMan addnode()
         throw std::runtime_error(
             self.ToString());
     }
+    const std::string manual_name{ConnectionTypeAsString(ConnectionType::MANUAL)};
+    const std::string manual_block_relay_name{ConnectionTypeAsString(ConnectionType::MANUAL_BLOCK_RELAY)};
+    if (strCommand == "remove" && !request.params[2].isNull()) {
+        throw JSONRPCError(RPC_INVALID_PARAMS, "Cannot specify a connection type with 'remove'");
+    }
+    std::string connection_type_name = manual_name;
+    if (!request.params[2].isNull()) {
+        connection_type_name = request.params[2].get_str();
+    }
+    ConnectionType connection_type;
+    if (connection_type_name == manual_block_relay_name){
+        connection_type = ConnectionType::MANUAL_BLOCK_RELAY;
+    }
+    else if (connection_type_name == manual_name) {
```

Comment:
clang-format says (also below)
```suggestion
    } else if (connection_type_name == manual_name) {
```

Reply:
done